autoload/lsp/signature.vim

```diff
@@ -57,7 +57,11 @@ export def BufferInit(lspserver: dict<any>)
 
   # map characters that trigger signature help
   for ch in lspserver.caps.signatureHelpProvider.triggerCharacters
-    exe $"inoremap <buffer> <silent> {ch} {ch}<C-R>=g:LspShowSignature()<CR>"
+    var mapChar = ch
+    if ch =~ ' '
```

Comment:
@pusewicz @yegappan maybe should use `==` not `=~` ? if like to try `space` as tgr char.